test/integration/template_admin_test.go

```diff
@@ -447,3 +447,22 @@ func TestLoadBalancerLegacyRenders(t *testing.T) {
 	}
 }
 
+func TestAdminEvictedServers(t *testing.T) {
+	// Path to the helm chart we will test
+	helmChartPath := testlib.ADMIN_HELM_CHART_PATH
+
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"admin.evicted.servers":   "{nuoadmin-1,nuoadmin-2}",
```

Comment:
I'm guessing the string `"{nuoadmin-1,nuoadmin-2}"` expand to a list. Is it possible to check that `nuoadmin-1` and `nuoadmin-2` are expanded as distinct entries in that list, rather than this expanding as a single string with a comma embedded inside of it?

Reply:
that is the limitation of the test. I don't like committing YAML files to git and using them in tests. The `{...,...} form is how you specify a list via the command line.

This is how a customer would specify it in real life:
```
$cat file.yaml 
admin:
  evicted:
    servers:
      - nuoadmin1
      - nuoadmin2
```

```
$ helm template --values ./file.yaml --show-only  templates/statefulset.yaml release-name stable/admin/
---
# Source: admin/templates/statefulset.yaml
apiVersion: "apps/v1"
kind: StatefulSet
...
        args:
          - "nuoadmin"
          - "--evicted-servers"
          - "nuoadmin1,nuoadmin2"
          - "--"
          - "pendingReconnectTimeout=60000"
          - "processLivenessCheckSec=30"
```

Reply:
What happens if you change the string to `"{ 'nuoadmin-1', 'nuoadmin-2' }"`? I'm not sure if that is valid Helm syntanx, but if that works it would be more convincing that we are actually combining distinct elements in a list to obtain `nuoadmin1,nuoadmin-2`.

Reply:
I think that the syntax used by Martin is the correct one to specify a list.
There is also an alternative syntax:
```
"admin.evicted.servers[0]": "nuoadmin1",
"admin.evicted.servers[1]": "nuoadmin2",
```